System/CLI/encryption.py

```diff
@@ -0,0 +1,43 @@
+import KeyUtils

```

Comment:
I believe that this should be "CLI.KeyUtils", as it's run from Main.py, not in the same directory. Additionally, is it possible for us to reformat this code as UpperCamelCase? I've been trying to make an effort to format my code that way, so to make sure it's all consistent, we should be using that. I apologize if that wasn't communicated clearly earlier.